### PR TITLE
[Docs] Update error_handler

### PR DESCRIPTION
**Pull request by jillnogold**
https://jira.iguazeng.com/browse/ML-3654

### CODE REVIEW

File: docs/serving/pipelines-error-handling.md [hunk 1]
```diff
@@ -1,39 +1,35 @@
...
-```
-graph2_enrich.error_handler("catcher")
-graph2.add_step("ErrHandler", name="catcher", full_event=True, after="")
-```
-```
-<mlrun.serving.states.TaskStep at 0x7fd46e557750>
+Example of an exception on a step that only runs when/if the "pre-process" step fails:
 ```
+graph = function.set_topology('flow', engine='async')
+graph.to(name='pre-process', handler='raising_step') .error_handler(name='catcher', handler='handle_error', full_event=True, before='echo')
```
Comment (davesh0812) [inline docs/serving/pipelines-error-handling.md:14]:
```suggestion
graph.to(name='pre-process', handler='raising_step').error_handler(name='catcher', handler='handle_error', full_event=True, before='echo')
```

File: docs/serving/pipelines-error-handling.md [hunk 2]
```diff
@@ -1,39 +1,35 @@
...
 
-Now, display the graph again:
-```
-graph2.plot(rankdir='LR')
+# Add another step after the error handling
```
Comment (davesh0812) [inline docs/serving/pipelines-error-handling.md:16]:
```suggestion
# Add another step after pre-process step or the error handling
```

File: docs/serving/use-cases.md [hunk 3]
```diff
@@ -100,29 +100,24 @@ Build and connect the graph (DAG) using the custom function and classes and plot
...
 
 Specify which step is the responder (returns the HTTP response) using the `step.respond()` method. 
 If the responder is not specified, the graph is non-blocking.
 
 ```python
-# use built-in storey class or our custom Echo class to create and link Task steps
+# use built-in storey class or our custom Echo class to create and link Task steps. Add an error handling step that runs only if the "Echo" step fails
 graph.to("storey.Extend", name="enrich", _fn='({"tag": "something"})') \
-     .to(class_name="Echo", name="pre-process", some_arg='abc').error_handler("catcher")
+     .to(class_name="Echo", name="pre-process", some_arg='abc') .error_handler(name='catcher', handler='handle_error', full_event=True)
```
Comment (davesh0812) [inline docs/serving/use-cases.md:111]:
```suggestion
     .to(class_name="Echo", name="pre-process", some_arg='abc').error_handler(name='catcher', handler='handle_error', full_event=True)
```

**Review by davesh0812 — APPROVED** [review]
Looks good some minor comments

File: docs/serving/realtime-pipelines.ipynb [hunk 4]
```diff
@@ -43,14 +43,14 @@
    "source": [
     "## The Event object\n",
     "\n",
-    "The Graph state machine accepts an Event object (similar to a Nuclio Event) and passes \n",
-    "it along the pipeline. An Event object hosts the event `body` along with other attributes \n",
-    "such as `path` (http request path), `method` (GET, POST, ..), and`id` (unique event ID).\n",
+    "The gaph state machine accepts an event object (similar to a Nuclio event) and passes \n",
```
Comment (alonmr) [inline docs/serving/realtime-pipelines.ipynb:46]:
```suggestion
    "The graph state machine accepts an event object (similar to a Nuclio event) and passes \n",
```

File: docs/cheat-sheet.md [hunk 5]
```diff
@@ -988,16 +988,19 @@ class ClassifierModel(mlrun.serving.V2ModelServer):
...
     kind="serving", image="mlrun/mlrun"
 )
 graph = function.set_topology("flow", engine="async")
 
-# use built-in storey class or our custom Echo class to create and link Task steps
+# use built-in storey class or our custom Echo class to create and link Task steps. Add an error 
+# handling step that runs only if the "Echo" step fails
 graph.to("storey.Extend", name="enrich", _fn='({"tag": "something"})') \
-     .to(class_name="Echo", name="pre-process", some_arg='abc').error_handler("catcher")
+     .to(class_name="Echo", name="pre-process", some_arg='abc') .error_handler(name='catcher', handler='handle_error', full_event=True)
```
Comment (alonmr) [inline docs/cheat-sheet.md:1003]:
```suggestion
     .to(class_name="Echo", name="pre-process", some_arg='abc').error_handler(name='catcher', handler='handle_error', full_event=True)
```

**Review by alonmr — COMMENTED** [review]
minor, and please also fix @davesh0812 's comments 🙂

**Pull request merged**